ambry-router/src/test/java/com.github.ambry.router/MockNetworkClient.java

```diff
@@ -38,14 +46,37 @@
   }
 
   /**
+   * {@inheritDoc}
+   */
+  MockNetworkClient(Selector selector, NetworkConfig networkConfig, NetworkMetrics networkMetrics,
+      int maxConnectionsPerPortPlainText, int maxConnectionsPerPortSsl, int checkoutTimeoutMs, Time time) {
+    super(selector, networkConfig, networkMetrics, maxConnectionsPerPortPlainText, maxConnectionsPerPortSsl,
+        checkoutTimeoutMs, time);
+  }
+
+  /**
    * Wake up the MockNetworkClient. This simply sets a flag that indicates that the method was invoked.
    */
   @Override
   public void wakeup() {
+    super.wakeup();
```

Comment:
Is this needed? This was not missed (as you can see from the javadoc of this method).